### PR TITLE
Fixed ru translation

### LINKED ISSUE #932: Opening any post throws Exception
I open http://127.0.0.1:8000/ru/blog/ and then when I try to click on any post header, I get the Exception.

image

### PR DESCRIPTION
**Pull request by yceruto**
Fix #932

### CODE REVIEW

**Comment by javiereguiluz** [comment]
I've been able to validate in my local app that this fixes the reported error, so let's merge it. Thanks Yonel.

File: translations/messages+intl-icu.ru.xlf [hunk 1]
```diff
@@ -306,7 +306,7 @@
             </trans-unit>
             <trans-unit id="post.num_comments">
                 <source>post.num_comments</source>
-                <target>{count, plural, one {# комментарий} few {# комментария} many {# комментариев}}</target>
+                <target>{count, plural, one {# комментарий} other {# комментариев}}</target>
```
Comment (stof) [inline translations/messages+intl-icu.ru.xlf]: this change is wrong. Russian has 3 differents forms. `few` should not have been removed

Comment (yceruto) [reply translations/messages+intl-icu.ru.xlf]: @stof The previous format doesn't work, what is the correct solution that you suggest?

Comment (stof) [reply translations/messages+intl-icu.ru.xlf]: well, `many` should have been `other`. But the `few` variant is still necessary for proper Russian pluralization AFAIK.

Comment (yceruto) [reply translations/messages+intl-icu.ru.xlf]: Got it! fixed in https://github.com/symfony/demo/pull/934

**Pull request merged**